modular_skyrat/master_files/code/game/turfs/closed/_closed.dm

```diff
@@ -0,0 +1,7 @@
+/turf/closed/indestructible/steel
```

Comment:
I'd rename this to this, for less confusion (yes you'll have to update it in your map too, sorry!
```suggestion
/turf/closed/indestructible/iron
```